packages/plugin-ext/src/plugin/languages.ts

```diff
@@ -77,7 +82,7 @@ export class LanguagesExtImpl implements LanguagesExt {
     private callId = 0;
     private adaptersMap = new Map<number, Adapter>();
 
-    constructor(rpc: RPCProtocol, private readonly documents: DocumentsExtImpl) {
+    constructor(rpc: RPCProtocol, private readonly documents: DocumentsExtImpl, private commands: CommandRegistryImpl) {
```

Comment:
I don't see where is used commands parameter ?

Reply:
property shorthand/parameter

Reply:
yes, read too fast, miss the private field